newrelic.yml

```diff
@@ -114,50 +121,50 @@ common: &default_settings
   # audit_log.path: log/newrelic_audit.log
 
   # An array of CLASS#METHOD (for instance methods) and/or CLASS.METHOD (for class
-  # methods) strings representing Ruby methods for the agent to automatically
-  # add custom instrumentation to without the need for altering any of the
+  # methods) strings representing Ruby methods that the agent can automatically
+  # add custom instrumentation to. This doesn't require any modifications of the
   # source code that defines the methods.
-  #
   # Use fully qualified class names (using the :: delimiter) that include any
   # module or class namespacing.
-  #
   # Here is some Ruby source code that defines a render_png instance method for an
   # Image class and a notify class method for a User class, both within a
   # MyCompany module namespace:
-  # 
+  # ``ruby
```

Comment:
@hannahramadan  - The docs team prefers if we add the language to code fences to help with rendering on the UI. I don't think the `newrelic.yml` generator template is prepared for this. I'll open an issue, but IIRC, you were the one who worked on it initially, so wanted to give you a heads up.

Reply:
Darn I forgot this was broken! We have this issue https://github.com/newrelic/newrelic-ruby-agent/issues/2866
Which I can edit to include the language part :◡̈